### PR TITLE
Tweaks on stacks-devnet-js lib

### PR DESCRIPTION
**Pull request by lgalabru**
This PR is adding a README to the stacks-devnet-js lib and fixing a deployment issue - the `dist` folder should be deployed to NPM, not the `lib`.

### CODE REVIEW

**Review by obycode — COMMENTED** [review]
It might be nice to add a bit more to this README so that someone with no background on this could follow along and use it. Or maybe just a link to the full tutorial video that you're working on would work as well.

File: node-bindings/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# stacks-devnet-js
+
+`stacks-devnet-js` is a node library, designed to let developers write integration tests for decentralized protocols built on top of the Stacks blockchain.
+It is implemented as a dynamic library that can be loaded by Node, and will let you orchestrate a Stacks Devnet network, locally, using Docker.
+
+### Installation
+
+```bash
+// Yarn
```
[the remark this answers is not in the document]
Comment (lgalabru) [reply node-bindings/README.md]: Addressed with https://github.com/hirosystems/clarinet/pull/181/commits/ea269aefc980b6aea663ff1115cec4321c13179c

File: node-bindings/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+yarn add dev @hirosystems/stacks-devnet-js
+
+// NPM
```
[the remark this answers is not in the document]
Comment (lgalabru) [reply node-bindings/README.md]: Addressed with https://github.com/hirosystems/clarinet/pull/181/commits/ea269aefc980b6aea663ff1115cec4321c13179c

File: README.md [hunk 3]
```diff
@@ -10,6 +10,21 @@ Blockstack. Smart contracts allow developers to encode essential business logic
 
 ![screenshot](docs/images/demo.gif)
 
+### Screencasts
+
+A serie of short tutorials is available as a playlist of screencasts on Youtube, covering the following subjects:
```
Comment (obycode) [inline README.md:15]: This should be "A series of".

File: node-bindings/README.md [hunk 4]
```diff
@@ -65,3 +65,11 @@ test('Block height changes when blocks are mined', async () => {
     console.log(`Next Block: ${JSON.stringify(block)}`);
 })
 ```
+
+### Screencasts
+
+A serie of short tutorials is available as a playlist of screencasts on Youtube, covering the following subjects:
```
Comment (obycode) [inline node-bindings/README.md:71]: This should be "A series of".

File: README.md (hunk 3 above)
Comment (lgalabru) [reply README.md:15]:
```suggestion
A series of short tutorials is available as a playlist of screencasts on Youtube, covering the following subjects:
```

File: node-bindings/README.md (hunk 4 above)
Comment (lgalabru) [reply node-bindings/README.md:71]:
```suggestion
A series of short tutorials is available as a playlist of screencasts on Youtube, covering the following subjects:
```

**Review by obycode — COMMENTED** [review]
Sorry, just noticed one more. Since you're already editing the README, I noticed it still says "designed by Blockstack" instead of Hiro.

**Review by obycode — APPROVED** [review]
👍

**Pull request merged**